src/FakeItEasy/Core/IFakeObject.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace FakeItEasy.Core
+{
+    /// <summary>
+    /// This interface is used internally by FakeItEasy and is exposed only due to technical constraints. It is not
+    /// intended to be used directly from your code.
+    /// </summary>

```

Comment:
Nice note. While we're at it, I wonder if a similar addition to `ITaggable` isn't a good idea.

Well, that's sort of what I wonder. What I really wonder is how I can convince everyone reading this that we should drop `ITaggable` in favour of `IFakeObject`.